Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM debian:buster-slim
+
+# Usage:
+# (1) Build image (only required once or after this file was changed):
+#     $ docker build -t photoboothbase .
+# (2) Start the container:
+#     $ docker run --rm -v "$(pwd):/var/www/html/" -p 8080:80 photoboothbase
+# (3) Go to: http://localhost:8080
+
+RUN set -x \
+    && apt-get update && apt-get dist-upgrade -y \
+    && apt-get dist-upgrade -y \
+    && apt-get install -y nginx php-fpm gphoto2 unclutter
```

Comment:
php-fpm includes gd? Or we also need php-gd?

Reply:
You're right - that was missing. Added it.